zypp/repo/SUSEMediaVerifier.cc

```diff
@@ -55,10 +56,16 @@ bool SUSEMediaVerifier::isDesiredMedia(const media::MediaAccessRef &ref)
   std::ifstream str(media_file.asString().c_str());
   std::string vendor;
   std::string id;
-#warning check the stream status
+
+  DBG << "reading data from repository media: " << media_file << endl;
```

Comment:
The above line is superfluous, as the location of the downloaded media_file is already reported in the log.